src/levels.rs

```diff
@@ -171,12 +171,12 @@ pub const FILTER_2D_8TAP_REGULAR_SHARP: Filter2d = 2;
 pub const FILTER_2D_8TAP_REGULAR_SMOOTH: Filter2d = 1;
 pub const FILTER_2D_8TAP_REGULAR: Filter2d = 0;
 
-pub type MVJoint = c_uint;
-pub const N_MV_JOINTS: usize = 4;
-pub const MV_JOINT_HV: MVJoint = 3;
-pub const MV_JOINT_V: MVJoint = 2;
-pub const MV_JOINT_H: MVJoint = 1;
-pub const _MV_JOINT_ZERO: MVJoint = 0;
+#[derive(Clone, Copy, PartialEq, Eq, FromRepr, EnumCount)]
+pub enum MVJoint {
```

Comment:
```suggestion
pub enum MVJoint {
    Zero = 0,
```

I think `EnumCount` is the count, not the max + 1.